torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -529,6 +529,45 @@ std::vector<ComputationClient::ComputationPtr> PjRtComputationClient::Compile(
   return computations;
 }
 
+bool PjRtComputationClient::SerializeComputation(ComputationPtr computation,
+                                                 std::ostream& out) {
+  XLA_CHECK(computation->devices().size() == 1 &&
+            computation->devices()[0] == "SPMD:0")
+      << "Persistent compilation cache only supports SPMD";
+  const PjRtComputation& pjrt_computation =
+      dynamic_cast<const PjRtComputation&>(*computation);
+
+  auto executable = pjrt_computation.executable->SerializeExecutable();
+  if (!executable.ok()) return false;
```

Comment:
Is it useful to log here if serialization fails?

Same for other cases where you check `ok`

Reply:
Oh good point, I didn't immediately remember how to use VLOG so I took the coward's way out (no logging) 😂